mapie/quantile_regression.py

```diff
@@ -318,7 +323,7 @@ def _check_cv(
         """
         if cv is None:
             return "split"
-        if cv == "split":
+        if (cv == "split") or (cv == "prefit"):
```

Comment:
Add "prefit" in documentation (line 306).

Reply:
done